.CI/build_all

```diff
@@ -11,7 +11,8 @@ if [ -z "$GH_TOKEN" ]; then
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
     conda install --yes --quiet conda-build-all
-    conda install --yes --quiet conda-build=1.20.0 jinja2 anaconda-client setuptools
+    conda install --yes --quiet conda-forge-build-setup
+    source activate root
```

Comment:
Let's skip the activate step, and just install a script from `conda-forge-build-setup` which we run. It wouldn't be such a problem if that script did stateful things like updating configs, or running other conda commands then.